tutorials/eda_contents/Computing_Environment/Unix.md

```diff
@@ -0,0 +1,363 @@
+
+On a Unix computer, the filesystem looks like something this:
+
+![The File System](https://cdn-wordpress-info.futurelearn.com/info/wp-content/uploads/a2794f8f-b0c1-468d-89c6-bcf29d2d6517-1.png)
+
+At the top is the **root directory** that holds everything else. We refer to it using a slash character `/` on its own;
+this is the leading slash in `/notebook_dir/writable-workspace/.home`.
+
+Inside that directory are several other directories:
+- `bin` (which is where some built-in programs are stored),
+- `lib` (for the software "libraries" used by different programs),
+- `home` (where users' personal directories are located),
+- `etc` (system-wide configuration files),
+and so on. Remember, everything is a file, so deleting files in a Unix environment, can have detrimental effects.
+
+Next, you may wish to explore the contents of a folder within the file system. We do this using the `ls`
+command hich stands for "listing":
+
+Note - in Marble, the basic `ls` command may yield no results 
+~~~
+$ ls
+
+$
+~~~
+
+This is because files may be hidden, to see these hidden files we the option `-a` or `--all` which
+gives us
+
+~~~
+$ ls -a
+
+.  ..  .bash_history  .ipython  .jupyter  .local  .npm
+~~~
+
+Note all files here start with `.`, this indicates they are hidden.
+
+`ls` has many other options. To learn about these options, we can used the `--help` modifier:
+
+~~~
+$ ls --help
+
+Usage: ls [OPTION]... [FILE]...
+List information about the FILEs (the current directory by default).
+Sort entries alphabetically if none of -cftuvSUX nor --sort is specified.
+
+Mandatory arguments to long options are mandatory for short options too.
+  -a, --all                  do not ignore entries starting with .
+  -A, --almost-all           do not list implied . and ..
+      --author               with -l, print the author of each file
+  -b, --escape               print C-style escapes for nongraphic characters
+      --block-size=SIZE      with -l, scale sizes by SIZE when printing them;
+                               e.g., '--block-size=M'; see SIZE format below
+   ...
+~~~
+
+
+To navigate through the filesystem, there is the `cd` command which stands for 'change directory'. Note, this commmand
+doesn't actually change the directory just the shell's idea of what directory we are in.
+
+~~~
+$ cd ..
+jovyan@97d07e419e83:/notebook_dir/writable-workspace$
+~~~
+
+Remember the hidden directory `..` from the `ls -a` command? 
+`..` is a special directory name meaning "the directory containing this one", or more succinctly,
+the **parent** of the current directory. In Marble, this command should take you to `/notebook_dir/writable-workspace`
+
+~~~
+$ pwd
+/notebook_dir/writable-workspace
+~~~
+
+Indicating the parent directory in Marble is `/notebook_dir/writable-workspace`.
+
+Using the `cd` command without an argument returns you to the home directory.
+
+We can also specify the path to the directory wish, for example:
```

Comment:
Is there a typo in: 

> to the directory wish 

(what is "directory wish"?)

Reply:
updated